JavaScript/es6Guidelines.md

```diff
@@ -46,9 +46,18 @@ We listed some useful tips to help you decide which declaration to use.
     shape = 'circle';
     ```
 
-2.  **Always use one let/const/var declaration in block scope**  
-    You should put lets and consts on top of the block.  
-    Even though consts and lets are not hoisted like vars it's a good practice to put those declarations on top of the block to increase code readability.
+2.  **Always use let/const declarations whenever you need it**
+    Consts and lets are not hoisted like vars - and you can use it multiple times in any given function - just whenever you need them. Consider following example:
+
+    ```javascript
+    const foo = this.get('foo');

```

Comment:
Let's make sure there's a "good" and "bad" example here (regardless of which guideline we go with).